src/libs/PolicyUtils.js

```diff
@@ -7,22 +7,28 @@ import ONYXKEYS from '../ONYXKEYS';
  * Checks if we have any errors stored within the POLICY_MEMBER_LIST. Determines whether we should show a red brick road error or not.
  * Data structure: {email: {role:'user', errors: []}, email2: {role:'admin', errors: [{1231312313: 'Unable to do X'}]}, ...}
  *
- * @param {Object} policyMemberList
+ * @param {Object} [policyMemberList]
  * @returns {Boolean}
  */
 function hasPolicyMemberError(policyMemberList) {
+    if (!policyMemberList) {
+        return false;
+    }
     return _.some(policyMemberList, member => !_.isEmpty(member.errors));
 }
 
 /**
  * Check if the policy has any errors, and if it doesn't, then check if it has any error fields.
  *
- * @param {Object} policy
+ * @param {Object} [policy]
  * @param {Object} policy.errors
  * @param {Object} policy.errorFields
  * @return {Boolean}
  */
 function hasPolicyError(policy) {
+    if (!policy) {
```

Comment:
For this one (and the one below), why is `policy` undefined?

Reply:
When we delete a policy here we set the value to null.
https://github.com/Expensify/App/blob/e3aab17009e6265615476e2687513b720f4e6570/src/libs/actions/Policy.js#L180

On the next render we pass a null policy to this function.

Reply:
Ah, OK. So, it's somewhat expected. I guess what you've done in this PR is OK, but it's a bit unexpected to have `null` in the list of policies. Another solution that cleans that up a little bit is in `AvatarWithIndicator` you could do:

```
// If a policy was just deleted from Onyx, then Onyx will pass a null value to the props, and 
// those should be cleaned out before doing any error checking
const cleanPolicies = _.compact(props.policies);
```
And then pass that to all the error checking. That will remove all the `null` values.

Reply:
oh, `compact()` is for arrays. Maybe `const cleanPolicies = _.filter(props.policies, policy => policy);` is what you need to keep it as an object. Though, it probably doesn't matter if it's cast to an array or not for the error checking.

Reply:
I think removing those which are null seems cleaner to me as well